lib/viewmodels/user_comments_view_model.dart

```diff
@@ -53,6 +54,8 @@ class UserCommentViewModel extends AutoDisposeAsyncNotifier<UserCommentsState> {
     // Refresh the home feed after comment deletion, that way the comment
     // count will be updated
     ref.read(postsFeedViewModelProvider.notifier).refresh();
+    // Also update the comment count for the post
+    ref.read(postCommentCountProvider(postId).notifier).refresh();
```

Comment:
You also mention this in the PR description, but why did you choose to refresh the count differently on comment deletion vs on refresh of a post's comments ?

Reply:
The difference is that, when we fetch the list of comment, we want to make sure there is no other online fetch and that the UI data stay consistent everywhere (i.e. that, if the user sees 3 comments under a post, the post is indeed marked to have 3 comments). There is no such optimisation needed in this case, since deletion does not fetch a comment count.
As mentionned in the previous comment, I however removed the `setCount` for more code uniformicity, to always trust Firestore's cache.

Reply:
ok it makes more sense now.